### PR TITLE
TOOL-985 Using new avd-manager step

### PR DESCRIPTION
**Pull request by lpusok**

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -36,27 +36,10 @@ workflows:
         - content: |-
             #!/bin/bash
             git clone -b master $SAMPLE_APP_URL .
-    - avd-manager:
+    - git::https://github.com/bitrise-steplib/steps-avd-manager@master:
```
Comment (godrei) [inline bitrise.yml:40]: pls comment why we lock for master

**Review by godrei — CHANGES_REQUESTED** [review]
pls check my note

File: bitrise.yml [hunk 2]
```diff
@@ -65,6 +48,7 @@ workflows:
         inputs:
         - test_to_run: Multiplatform.UItest.Tests(Android)
         - build_tool: msbuild
+        - emulator_serial: "emulator-5554"
```
Comment (godrei) [inline bitrise.yml]: do we need this? the avd manager exposes the emulator serial and that env should be the default value of this input

File: bitrise.yml (hunk 1 above)
Comment (lpusok) [reply bitrise.yml:40]: added comment

**Review by godrei — APPROVED** [review]

**Pull request merged**